easybuild/easyconfigs/p/PSolver/PSolver-1.7.6-foss-2017b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'PSolver'
+version = '1.7.6'
+
+homepage = 'http://bigdft.org/Wiki/index.php?title=BigDFT_website'
+description = """Poisson Solver from the BigDFT code compiled as a standalone library."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'usempi': True, 'opt': True}
+
+source_urls = ['https://launchpad.net/bigdft/1.7/%(version)s/+download/']
+sources = ['bigdft-%(version)s.tar.xz']
+checksums = ['cdb443380540eb3238c5393413f031dfa3a649c18edf5212433a06e19b03929c']
+
+dependencies = [
+    ('libyaml', '0.1.7'),
```

Comment:
@micaeljtoliveira This'll need to be bumped to 0.2.1 in all 3 easyconfigs, see failing dep variant test.

And we should consider trashing the `libyaml` easyconfig you added in #6887 I guess, since there's little point in having it around (I should have noticed that before merging the PR).

Reply:
@boegel I've bumped `libyaml` dependency to 0.2.1 for the foss/2018b toolchain. To do the same for foss/2017b and foss/2018a I would need to add libyaml-0.2.1 to those toolchains. I just want to be sure this is what is intended.